Header header

geometry_msgs/Vector3 foot_0
geometry_msgs/Vector3 foot_1
geometry_msgs/Vector3 foot_2
geometry_msgs/Vector3 foot_3